### ISSUE TITLE
XceptionTimePlus not working using TSForecaster

### ISSUE THREAD

**Issue by MOREDataset** [timeseriesAI/tsai#527]
Hi @oguiza, I get an error when I try to use the XceptionTImePlus for multi-step forecasting.
I tried a simple code as well and it still gives me the same error:
`learn = TSForecaster(X_train, y_train, splits=splits, tfms=tfms, bs=512,
                      arch=XceptionTimePlus, arch_config={},
                      metrics=rmse)# , cbs=FastAIPruningCallback(trial))
`

Error found:
![image](https://user-images.githubusercontent.com/14005584/170599961-4f2eac1a-1918-4836-bbbc-cf73a48095c9.png)

**Comment by oguiza** [comment]
Hi @MOREDataset,
Sorry for my late reply.
I believe this issue is fixed now. I've run the following code and don't get the issue you mention: 
```python
X, y, splits = get_UCR_data('LSST', split_data=False)
tfms = [None, TSClassification()]
learn = TSForecaster(X, y, splits=splits, tfms=tfms, bs=512, arch=XceptionTimePlus, arch_config={})
learn.fit_one_cycle(1)
```

**Comment by parshinsh** [comment]
getting the same error for the MiniRochetPlus architecture in TSForecaster.

> Hi @MOREDataset, Sorry for my late reply. I believe this issue is fixed now. I've run the following code and don't get the issue you mention:
> 
> ```python
> X, y, splits = get_UCR_data('LSST', split_data=False)
> tfms = [None, TSClassification()]
> learn = TSForecaster(X, y, splits=splits, tfms=tfms, bs=512, arch=XceptionTimePlus, arch_config={})
> learn.fit_one_cycle(1)
> ```

**Comment by mPowc** [comment]
这是来自QQ邮箱的假期自动回复邮件。您的邮件已收到，谢谢！

**Comment by MOREDataset** [comment]
Hey there @oguiza, thank you for your efforts but I just tried the model again for time series forecasting and got the same error.

**Comment by oguiza** [comment]
Hi @MOREDataset , @parshinsh, @mPowc ,
I've found custom_head was actually missing in 3 models: MiniRocketPlus, XceptiontimePlus, and FCNPlus. The issue is fixed now (#597).
